model_compression_toolkit/gptq/common/gptq_training.py

```diff
@@ -261,11 +207,9 @@ def build_gptq_model(self):
                              f'framework\'s GPTQ model builder method.')  # pragma: no cover
 
     @abstractmethod
-    def train(self, representative_data_gen: Callable):
+    def train(self):
         """
-        Train the quantized model using GPTQ training process
-        Args:
-            representative_data_gen: Dataset to use for inputs of the models.
+        Train the quantized model using GPTQ training process.
         """
         raise NotImplemented(f'{self.__class__.__name__} have to implement the '
```

Comment:
NotImplementedError? (;

Reply:
I didn't write this line!

Reply:
I know, but you also changed many other such exceptions in places you modified other code (like in the fw implementation) so I just wondered why here you decided to let it go 😄

Reply:
I didn't decide, I just didn't notice it. Probably losing sight due to old age, that's the only possible explanation

Reply:
that's what I feared of 😆